assets/queries/openAPI/parameter_object_incorrect_ref/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	params := doc.paths[name].parameters[n]
```

Comment:
Parameters can be defined in other attributes - https://swagger.io/docs/specification/describing-parameters/#path-parameters. Will it be relevant to consider?